README.md

```diff
@@ -129,6 +133,28 @@ class MyProcessor
 end
 ```
 
+### Statsd integration
+
+You can pass a `statsd_client` to the `GovukMessageQueueConsumer::Consumer` initializer. The consumer will notify statsd with these keys:
```

Comment:
Perhaps mention stats are of counter type e.g. 'will emit counters to statsd with these keys'